people/salary.md

```diff
@@ -4,48 +4,41 @@
 
 Because we live in different parts of the world, have different lifestyles and different tax laws it's almost impossible for the company to set a fair salary for everyone. Therefore, **we give the biggest weight to your own proposal** for your salary. The founders' salaries are published below for orientation.
 
-We have a salary cap which is five times that of the current minimum salary in the company because we hate how CEOs of some corporations [earn hundreds of times more] than their minimum wage workers.
+We have a salary cap which is five times that of the current minimum salary in the company because we hate how CEOs of some corporations [earn hundreds of times more](https://www.cnbc.com/2018/01/22/heres-how-much-ceo-pay-has-increased-compared-to-yours-over-the-years.html) than their minimum wage workers.
 
-As a company, we made the decision ([IRL#4]) to have an open salary policy in order to promote equality and transparency. There is no overall list for quick comparison instead the salary and bonuses of each Nitean are available in our Scrooge salary app to be viewed on an individual basis.
+As a company, we made the decision ([IRL#4](https://blog.niteo.co/irl4-istanbul/)) to have an open salary policy in order to promote equality and transparency. There is no overall list for quick comparison instead the salary and bonuses of each Nitean are available in our Scrooge salary app to be viewed on an individual basis.
 
 We welcome questions about salaries, bonuses, and responsibilities to help you decide what would be a fair salary. Everyone is also free to propose that someone should have a higher/lower salary or that we should increase minimum salary (which is the only way to increase the maximum salary).
 
+If you live in a country that has their economy tied to USD more than EUR, it is recommended that you choose to receive your salary in USD in order to decrease the risk of currency fluctuations affecting your base salary.
 
-## Current company salaries (June 2018)
+### Current company salaries (July 2018)
 
 All amounts are gross (total cost to the company). Some people take care of their own taxes and for some the company pays them, however the gross salary is the same.
 
 **Minimum salary:** 1300 EUR
 
-**Founders' salary:** 2500 EUR (seniors are paid more)
+**Salary cap:** 6500 EUR (factor of 5)
 
-**Salary cap:** 6500 EUR ([factor of 5])
 
+### What to consider when setting your salary?
 
-## What to consider when setting your salary?
-
-The main idea to keep in mind is: base salary gives you a *bit painful* lower-middle-class salary that covers your expenses while [profit sharing](profit-sharing.md) gives you the luxury and brings you in the higher class. We want the large majority of salary to come from profit sharing so that the company is able to survive challenging times with as low fixed expenses as possible, but so that everyone gets a big cut of the profit when we're doing well.
```

Comment:
I'm against removing this until we have a better recommendation / base idea on how to decide for a salary.

Reply:
Added a new section.